careplan.js

```diff
@@ -923,33 +995,12 @@ function waitCarePlan() {
       function handleCarePlanTabClick() {
         if (location.href.includes("all_plans")) {
           if (healthieAPIKey !== "") {
-            cpTabContents && cpTabContents.empty();
+            //cpTabContents && cpTabContents.empty();

```

Comment:
are you planning to reuse this if? if not whole if statement could be deleted

Reply:
I removed the if statement, it was left over from testing